web/src/components/storage/zfcp/ZFCPDiskForm.tsx

```diff
@@ -107,7 +107,7 @@ export default function ZFCPDiskForm({
   if (!formData.channel && getChannels().length > 0) select();
 
   return (
-    <Page.Section>
+    <Page.Section aria-label="zfcp-activation-form">
```

Comment:
This is not a valid aria-label.nit looks more like an id.  aria-label should be a human readable text. Don't forget to mar it for translation.